flake.nix

```diff
@@ -66,6 +59,7 @@
           pkgs.tigerbeetle
           self.packages.${system}.libtb_client
         ];
+        nativeBuildInputs = [ pkgs.pkgconf ];
```

Comment:
`pkg-config` isn't as actively maintained as `pkgconf`

Reply:
will this actually make it available for includes?